indexer/src/main/java/au/org/aodn/esindexer/controller/IndexerController.java

```diff
@@ -48,11 +48,22 @@ public ResponseEntity<ObjectNode> getDocumentByUUID(@PathVariable("uuid") String
         ObjectNode response =  indexerService.getDocumentByUUID(uuid).source();
         return ResponseEntity.status(HttpStatus.OK).body(response);
     }
-
+    /**
+     * A synchronized load operation, useful for local run but likely fail in cloud due to gateway time out. No response
+     * come back unlike everything done. Please use async load with postman if you want feedback constantly.
```

Comment:
is that this one? https://blog.postman.com/send-asynchronous-requests-with-postmans-pm-api/

Reply:
No as the function will not return anything until all ingest complete, hence it will result in gateway timeout in the AWS